src/index.js

```diff
@@ -4,7 +4,24 @@
  */
 
 const pascalTriangle = (lineNumber) => {
-  
+
+  if (lineNumber == 0) {
+      let fila = [1];
+      return fila;
+  }else{
+      let fila = [0, 1, 0];
+      let nuevaFila = [];
```

Comment:
No usas `nuevaFila` fuera del for, por lo que puedes declararlo dentro con block scope y no sería necesario restearla como en la linea 21